src/interaction/interactiveTarget.js

```diff
@@ -42,5 +49,11 @@ module.exports = {
      * @todo Needs docs.
      * @private
      */
-    _touchDown: false
+    _touchDown: false,
+    
+    /**
+     * Used to track the global start position of a touchdown event on this target.
+     * @private
+     */
+    _touchDownPoint: new core.Point()
```

Comment:
Pretty sure this object reference will be shared among all instances, see:
https://github.com/naranjamecanica/pixi.js/blob/dev/src/interaction/InteractionManager.js#L5-L8

Is this ok?